tests/e2e/docusaurus2-graphql-doc-generator-multi-instance.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = [
+  {
+    id: "config-A",
+    schema: "data/tweet-A.graphql",
+    rootPath: "./docs-tweet-A",
+    baseURL: "schema",
+    linkRoot: "/",
+    loaders: {
+      UrlLoader: "@graphql-tools/url-loader",
+    },
+  },
+  {
+    id: "config-B",
+    schema: "data/tweet-B.graphql",
+    rootPath: "./docs-tweet-B",
+    baseURL: "schema",
+    linkRoot: "/",
+    loaders: {
+      UrlLoader: "@graphql-tools/url-loader",
+    },
+  },
```

Comment:
Same as above
```suggestion
  {
    id: "tweet",
    schema: "data/tweet.graphql",
    rootPath: "./docs-tweet",
    baseURL: "schema",
    linkRoot: "/",
  },
  {
    id: "swapi",
    schema: "data/swapi.graphql",
    rootPath: "./docs-swapi",
    baseURL: "schema",
    linkRoot: "/",
  },
```

This can also be simplified to:
```js
[
  require('./docusaurus2-graphql-doc-generator-multi-instance-1.config.js'),
  require('./docusaurus2-graphql-doc-generator-multi-instance-2.config.js'),
]
```

Reply:
Nice refactoring. Addressed in 9b550e2.